### PR TITLE
[Ready] Atmos cleanup + subtypes

### PR DESCRIPTION
**Pull request by 81Denton**
I cleaned up a bunch of atmos related dvs and created subtypes that should make mapping atmospherics easier and faster.

- Igniters now default to on = FALSE - this makes sense since they start switched off in absolutely all maps except for CentCom and two away missions. I removed the luminosity overrides as well, since igniter luminosity is defined by the power state.
- added mixer/airmix subtype that uses N2STANDARD/O2STANDARD and MAX_OUTPUT_PRESSURE instead of "0.8/0.2" and "4500"
- added freezer/on and heater/on, with the freezer defaulting to the lowest possible temp
- added waste loop specific filter/atmos subtypes (filter/atmos/o2 etc.)

### CODE REVIEW

**Comment by ShizCalev** [comment]
Hold on this.

**Comment by Cyberboss** [comment]
@MapDiffBot

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm [hunk 1]
```diff
@@ -166,6 +166,10 @@
 	max_temperature = T20C
 	min_temperature = 170
 	circuit = /obj/item/circuitboard/machine/thermomachine/freezer
+	
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer/on
+	on = TRUE
+	target_temperature = 73.15
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: line 167 `min_temperature = 170`?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: Freezers with stock parts cool up to 73.15K, even though min_temperature is 170K. No clue why, maybe RefreshParts() causes it?

**Review by duncathan — DISMISSED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: Maps used to have roundstart target_temp defined as 80K, doesn't make much sense imo when the default mininum temp is 73.15K.

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm [hunk 2]
```diff
@@ -164,28 +164,37 @@
 	icon_state_on = "freezer_1"
 	icon_state_open = "freezer-o"
 	max_temperature = T20C
-	min_temperature = 170
+	min_temperature = 170 //actual minimum temperature is defined by RefreshParts()
 	circuit = /obj/item/circuitboard/machine/thermomachine/freezer
+	
+/obj/machinery/atmospherics/components/unary/thermomachine/freezer/on
+	on = TRUE
+	target_temperature = 73.15
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: Change this to Init and do `target_temperature = min_temperature` instead, that way it'll be set to the correct temp if a mapper modifies the parts.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm (hunk 2 above)
Comment (81Denton) [reply code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: I like that idea! Let me change the files.

Comment (81Denton) [reply code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm]: Okay done

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**